docs/conf.py

```diff
@@ -69,7 +69,7 @@
     "pandas": ("https://pandas.pydata.org/pandas-docs/stable/", None),
     "matplotlib": ("https://matplotlib.org/", None),
     "traitlets": ("https://traitlets.readthedocs.io/en/stable/", None),
-    "ctapipe": ("https://ctapipe.readthedocs.io/en/v0.18.1/", None)
+    "ctapipe": ("https://ctapipe.readthedocs.io/en/v0.19.2/", None)
```

Comment:
what about stable?

Reply:
#1122

Reply:
Not a good idea I think. It will likely break our docs build as soon as a new ctapipe version is released and we don't immediately update.

Reply:
OK, I see. Then maybe worth checking if we need to pin other packages versions (I see e.g. python 3.6)

Reply:
decided not to go for it after [the discussion](https://github.com/cta-observatory/cta-lstchain/pull/1122/files/fe15f07b33a116c98c153f58e006fd8a5afa840b#r1231908329)